### PR TITLE
#159460710 Remove serial attribute from asset model

### PR DESCRIPTION
**Pull request by Victoradukwu**

### CODE REVIEW

File: tests/mocks/asset.py [hunk 1]
```diff
@@ -1,55 +1,42 @@
...
 A set of mock asset data for testing asset creation
 """
-ASSET_TWO = {
-    "tag": "Fred Macbook",
-    "serial": "sbgdvt4528j8gg",
-    "customAttributes": {}
-}
+ASSET_TWO = {"tag": "Fred Macbook", "customAttributes": {}}
 ASSET_THREE = {
     "tag": "King Macbook",
```
Comment (GbengaOshinaga) [inline tests/mocks/asset.py]: Remove trailing comma

File: tests/mocks/asset.py [hunk 2]
```diff
@@ -1,55 +1,40 @@
...
-    "tag": "Fred\'s Macbook",
-    "serial": "sbgdvt4528j8gg",
-    "customAttributes": {}
-}
+ASSET_EMPTY_CUSTOM_ATTRS = {"tag": "Fred\'s Macbook", "customAttributes": {}}
 
-ASSET_NO_TAG = {"serial": "sbgdvt4528j8hg"}
+ASSET_NO_TAG = {"tag": ""}
 
 ASSET_NO_SERIAL = {"tag": "Jude\'s Macbook"}
```
Comment (GbengaOshinaga) [inline tests/mocks/asset.py]: @Victoradukwu This `ASSET_NO_SERIAL` should be deleted since it is no longer being used

**Comment by Victoradukwu** [comment]
@GbengaOshinaga All requested changes have been implemented.

**Comment by rogerokello** [comment]
This is great work, @Victoradukwu.  Just wanted a bit of clarity regarding something small. In the section `How should this be manually tested?` under the second bullet, what did you mean by Run `purest`?

**Comment by Victoradukwu** [comment]
@rogerokello Thanks for spotting that. The typo has been fixed.

**Comment by rogerokello** [comment]
LGTM

**Comment by SEUNAGBEYE** [comment]
@Victoradukwu Please resolve merge conflicts.

**Pull request closed without merging**